lektor/admin/static/js/widgets/ToastEditor.jsx

```diff
@@ -0,0 +1,213 @@
+'use strict'
+
+import React from 'react'
+import Editor from 'tui-editor'
+
+import Component from '../components/Component'
+
+import 'codemirror/lib/codemirror.css'
+import 'tui-editor/dist/tui-editor.min.css'
+import 'tui-editor/dist/tui-editor-contents.min.css'
+
+import LektorLinkExtension from './tuiEditorLektorLinkExt.jsx'
+import LektorImageExtension from './tuiEditorLektorImageExt.jsx'
+
+const toolbarItems = [
+  'heading',
+  'bold',
+  'italic',
+  'strike',
+  'divider',
+  'hr',
+  'quote',
+  'divider',
+  'ul',
+  'ol',
+  // 'task',
+  'indent',
+  'outdent',
+  'divider',
+  'table',
+  // 'image',
+  // 'link',
+  'divider',
+  'code',
+  'codeblock'
+]
+
+class ToastEditor extends Component {
+  constructor () {
+    super()
+    this.element = null
+    this.count = 0
```

Comment:
What is the purpose of this attribute? I can see that it's used in `onChange` but I don't quite understand the purpose of it. If this is needed for something important it should either have a better name or a comment explaining its purpose.

Reply:
This was used to workaround a bug in the editor where onChange was fired as soon as the editor was created and had no content, but has since been superseded by better handling of this behaviour. This has now been removed.